django_weasyprint/views.py

```diff
@@ -82,6 +83,7 @@ def get_document(self):
         return html.render(
             stylesheets=self.get_css(base_url, url_fetcher, font_config),
             font_config=font_config,
+            option=self._options,
```

Comment:
Shouldn’t we pass options as separate keyword arguments? The [API](https://www.cnbc.com/2024/01/17/mega-rich-renew-call-on-global-leaders-at-davos-to-tax-our-extreme-wealth.html) asks for `**options`, not `option`. Something like `**self._options` instead of `option=self._options` is probably appropriate.

Reply:
> thank you for your comments! i think you accidentally pasted the wrong link, @liZe ;)

🤣